tests/ui/libraries/test_libraries_api.py

```diff
@@ -153,3 +154,20 @@ def test_library_timezone(lib_martigny):
     """Test library timezone."""
     tz = lib_martigny.get_timezone()
     assert tz == pytz.timezone('Europe/Zurich')
+
+
+def test_library_get_address(lib_martigny, lib_saxon):
+    """Get information about a library address."""
+    lib = lib_martigny
+    address = lib.get_address(LibraryAddressType.MAIN_ADDRESS)
+    assert address == lib.get('address')
+    address = lib.get_address(LibraryAddressType.SHIPPING_ADDRESS)
+    assert address['country'] == 'ch'
```

Comment:
if someone changes the `data.json` this test will not work any more.

Reply:
Yes :-) Pay attention when you update unitest fixtures :-)